jobs/config.json

```diff
@@ -318,6 +318,25 @@
       "sig-apps"
     ]
   },
+  "spark-k8s-periodic": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gke.env",
+      "--extract=ci/latest",
+      "--gcp-node-image=gci",
+      "--gcp-zone=us-central1-f",
+      "--provider=gke",
+      "--test=false",
+      "--timeout=180m",
+      "--gke-environment=prod",
+      "--test-name=spark-integration",
+      "--test-cmd='cd spark-integration && ./e2e/e2e-prow.sh'"
```

Comment:
I think the file tree looks like
```
workspace
- spark-integration
  - (stuff inside spark-integration repo)
  - kubernetes
```

Reply:
It wouldn't land me within `workspace/`?

Reply:
I believe `--extract` will cd into kubernetes